src/components/WithTrackingHOC/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import ReactGA from 'react-ga'
+
+interface WithTrackerHOCProps extends Window {}
+
+export default function withTracker(WrappedComponent: React.SFC | React.ComponentClass, options = {}) {
```

Comment:
No need to wrap every component on every route. This can be just one component rendered on every route, it only needs to have access to Router props.
Like:
```
export default withRouter(class extends Component<WithTrackerHOCProps> {
...
```

And put it below `<RedirectToDisclaimer/>`

Reply:
is that better? more efficient or just DRY? Just trying to undershtand

Reply:
all of them. With HOC you'll have 
```
<Wrapper>
 <Component>
</Wrapper>
```
two components for one functionality with parent retriggering child render every time

With one component always there in the tree and if it also doesn't render anything, impact is minimal